commands.go

```diff
@@ -33,7 +33,7 @@ var (
 		{"hang", hangCMD, "<char|word>", "Play hangman"}, // won't actually run, here just to show in docs
 		{"tic", ticCMD, "<cell num>", "Play tic tac toe!"},
 		{"cd", cdCMD, "#room/user", "Join #room, DM user or run cd to see a list"}, // won't actually run, here just to show in docs
-		{"tz", tzCMD, "<zone>", "Set your IANA timezone (like tz Asia/Dubai)"},
+		{"tz", tzCMD, "<zone> [format]", "Set your IANA timezone (like tz Asia/Dubai). Set format to 24 to have hour in the 0-23 range."},
```

Comment:
Change format to 24 and shorten that help message to “set your IANA ... and set 24h clock

Reply:
Good idea, I was wondering how I could shorten the help message.